radlibDemoApp/www/js/index.js

```diff
@@ -53,7 +53,7 @@ function getParsed() {
    var selectedReader = dropdown.options[dropdown.selectedIndex].value;
 
    if (selectedReader == "scan") {
-      radlib.directConnect(dumpLog, dumpLog, "", "PARSED"); // CURRENTLY BROKEN
+      radlib.scanReaders(deleteMe, dumpLog, ["BLUETOOTH"]);

```

Comment:
because deleteMe actually parses the information. It returns an array of objects. Had I called dumpLog, it'd be "Object object" showing up, which wouldn't mean much. deleteMe is so that other ppl can see what I'm returning and how to access.